operational/vmware/instance_tag_sync/instance_tag_sync.pt

```diff
@@ -0,0 +1,426 @@
+name "VMWare Instance Tag Sync"
+rs_pt_ver 20180301
+type "policy"
+short_description "Adds tags to vmware instances from CMP. See the [README](https://github.com/rightscale/policy_templates/tree/master/operational/vmware/instance_name_sync) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+severity "medium"
+category "Operational"
+
+###############################################################################
+# Permissions
+###############################################################################
+
+permission "perm_read_creds" do
+  actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
+  resources "rs_cm.credentials"
+end
+
+permission "perm_retrieve_resources" do
+  label "Retrieve Resources"
+  actions   "rs_cm.index","rs_cm.show"
+  resources "rs_cm.instances","rs_cm.volumes","rs_cm.clouds"
+end
+
+permission "perm_tags" do
+  label "Retrieve and add Tags"
+  actions "rs_cm.by_resource","rs_cm.multi_add"
+  resources "rs_cm.tags"
+end
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify"
+end
+
+parameter "param_cloud_name" do
+  type "string"
+  label "Cloud Name"
+end
+
+parameter "param_wstunnel_token" do
```

Comment:
I would think this is better stored as a cred, could prompt for the credentials name.

Reply:
Since it's possible to have multiple vmware clouds in a single account for MVP I would like to leave them as inputs so you don't have to clone it a bunch.

Reply:
Fair point, but you could also just prompt for the credential name.